### ISSUE TITLE
Modern user interface

### ISSUE THREAD

**Issue by ShockingArtist** [Team-MCToolkit/MCToolkit#111]
**Describe the feature**

Modern user interface for MCtoolkit

**How would you implement it?**

Mockup: https://www.figma.com/file/nlpocq6hKLBRsr0LyyuF73/Untitled?node-id=0%3A1
This is going to be a huge change so I suggest we implement in a later big release

**Comment by ShockingArtist** [comment]
#96

**Comment by ShockingArtist** [comment]
<img width="182" alt="Screen Shot 2020-11-30 at 3 06 02 PM" src="https://user-images.githubusercontent.com/65963376/100658745-90b85600-331d-11eb-9110-7e8c698c9e81.png">
Folder Test

**Comment by ShockingArtist** [comment]
Changed Futura to SF Pro display for license issues

**Comment by dujinhuachinaeddie** [comment]
wow you're a bunch of geniuses!

**Comment by willpill** [comment]
Just finished Mockup No.2 Link added in post

**Comment by willpill** [comment]
Updating the new icon for Big Sur
![newicon](https://user-images.githubusercontent.com/65963376/106612962-ad4c5680-6537-11eb-9ef7-392c6181401d.png)
Would love for someone to test

**Comment by Max094Reikeb** [comment]
No, the "current" icon is better. Windows icon is flat, so should MacOS one stay.

**Comment by willpill** [comment]
They are supposed to be different styles. The "current" MacOS icon does not fit

**Comment by Max094Reikeb** [comment]
What do you mean by "does not fit"?

**Comment by willpill** [comment]
https://developer.apple.com/design/human-interface-guidelines/macos/icons-and-images/app-icon/

**Comment by willpill** [comment]
Oh and also FYI *if* the icon is accepted it will be pushed when the UI&UX update is complete

**Comment by Max094Reikeb** [comment]
Who care? A lot of apps have icons that "does not fit" in MacOS : IntelliJ, Visual Studio Code, GitHub Desktop, Brave Browser, ...

**Comment by willpill** [comment]
https://github.com/desktop/desktop/pull/11388 Github Desktop is being updated

**Comment by willpill** [comment]
https://github.com/microsoft/vscode/issues/101014 VSC icon is being changed

**Comment by willpill** [comment]
and etc. Visual icons are the first thing users see so it's very important

**Comment by Max094Reikeb** [comment]
"important" very good icons have been proposed for VSCode but it has been since June 2020 and still no PR to close the Issue.

**Comment by willpill** [comment]
Yeah because there are so many choices and... it's microsoft

**Comment by willpill** [comment]
<img width="86" alt="Screen Shot 2021-02-02 at 9 40 06 AM" src="https://user-images.githubusercontent.com/65963376/106615667-ab37c700-653a-11eb-8199-216ef74ce34b.png">
There I tested it myself
